<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgb(251, 162, 208), stop:1 rgb(255, 196, 164));


</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 30px;
background-color:rgb(108, 126, 225);
color: white ;
border-radius: 10px</string>
     </property>
     <property name="text">
      <string>Calendar</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">#calendarWidget QWidget {
background-color: rgb(146, 185, 227);
color: black;
border-radius:5px;
}

#qt_calendar_monthbutton{
	width : 130px;
	font-size : 23px;
	margin : 5px 1px;
}

#qt_calendar_yearbutton
{
	width : 90px;
	font-size : 23px;
	margin : 5px 1px;
}

#qt_calendar_monthbutton:hover,
#qt_calendar_yearbutton:hover,
#qt_calendar_prevmonth:hover,
#qt_calendar_nextmonth:hover
{
	background-color:rgb(255, 117, 71)
}

#qt_calendar_monthbutton:pressed,
#qt_calendar_yearbutton:pressed,
#qt_calendar_prevmonth:pressed,
#qt_calendar_nextmonth:pressed{
	background-color:rgb(255, 117, 71)
}</string>
       </property>
       <property name="firstDayOfWeek">
        <enum>Qt::Monday</enum>
       </property>
       <property name="gridVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="taskLineEdit">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 10px;
background-color:white</string>
         </property>
         <property name="placeholderText">
          <string>Write Plan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="groupComboBox">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#groupComboBox {
	padding-left : 10px
}
</string>
         </property>
         <item>
          <property name="text">
           <string>Important, Urgent</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Import, Not Urgent</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Not Important, Urgent</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Not Important, Not Urgent</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(198, 136, 235);
	border-radius: 10px;
}
QPushButton:hover{ 
	background-color: rgb(195,81,167);
}</string>
         </property>
         <property name="text">
          <string>Add new</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="tasksListWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(198, 136, 235);
	border-radius: 10px;
}
QPushButton:hover{ 
	background-color: rgb(195,81,167);
}</string>
         </property>
         <property name="text">
          <string>Delete task</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(198, 136, 235);
	border-radius: 10px;
}
QPushButton:hover{ 
	background-color: rgb(195,81,167);
}</string>
         </property>
         <property name="text">
          <string>Save change</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="plotButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(198, 136, 235);
	border-radius: 10px;
}
QPushButton:hover{ 
	background-color: rgb(195,81,167);
}</string>
         </property>
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
